### ISSUE TITLE
Can you rename elf2 BudgieLoader.exe to BudgieLoader2.exe ?

### ISSUE THREAD

**Issue by nixxou** [teknogods/TeknoParrotUI#907]
The reason is, as stupid as it is, in amd control panel, you can't set a custom profile for C:\teknoparrot\ElfLdr2\BudgieLoader.exe if you already have one for C:\teknoparrot\TeknoParrot\BudgieLoader.exe you need to have a different filename for the executable.
If not, the profile will only apply to the old BudgieLoader, and the new one will always use global settings.

And i know, it's amd fault, in fact this is so stupid that i double checked on discord and get confirmation from other users : 
![image](https://github.com/user-attachments/assets/94497326-8807-47c4-93d1-79512d0c4adf)

**Comment by antoniodesousa** [comment]
Yes, I also want this request to be fulfilled! It has been a very annoying bug on AMD side for years.
And renaming the executable to BudgieLoader2.exe can be that hard, right?
